h/accounts/models.py

```diff
@@ -132,6 +133,11 @@ def is_activated(self):
 
         return False
 
+    def activate(self, request):
+        """Activate the user by deleting any activation they have."""
+        request.db.delete(self.activation)
+        request.registry.notify(events.ActivationEvent(request, self))
```

Comment:
And a logic module is out of the question? I guess you could argue that just the one thing isn't enough to make a logic module worth it. But otherwise, _that_ seems like the right abstraction to me.

(Have to admit I'm not seeing what's definitely wrong about adding one param and one line to the model class, that seems equally as wrong as two lines and two tests of duplication to me.)